client/state/data-layer/wpcom/sites/stats/google-my-business/index.js

```diff
@@ -16,54 +16,55 @@ import { registerHandlers } from 'state/data-layer/handler-registry';
 
 export const fromApi = data => convertToCamelCase( data );
```

Comment:
Here we could remove the arrow function and what remains is:
```js
export const fromApi = convertToCamelCase;
```
Then, if we stop testing `fromApi`, we don't need to export it, and can just pass `convertToCamelCase` directly as
```js
dispatchRequestEx( {
  fromApi: convertToCamelCase
} );
```